crates/rolldown/tests/rolldown/sourcemap/live_binding/artifacts.snap

```diff
@@ -51,24 +51,24 @@ Object.defineProperty(exports, 'shared', {
 
 ```
 - ../main1.js
-(2:0-2:9) "\nfunction" --> (4:0-4:9) "\nfunction"
-(2:9-2:16) " main()" --> (4:9-4:16) " main()"
-(2:16-3:2) " {\n " --> (4:16-5:0) " {"
-(3:2-0:0) "" --> (5:0-5:1) "\n"
-(0:0-3:2) "import {shared} from './shared.js'\n\nfunction main() {\n " --> (5:1-5:16) "\trequire_shared"
-(3:2-3:10) " shared(" --> (5:16-5:24) ".shared("
-(3:10-4:1) ");\n" --> (5:24-6:0) ");"
-(4:1-6:0) "}\n" --> (6:0-7:0) "\n}"
-(6:0-6:6) "\nmain(" --> (7:0-7:6) "\nmain("
-(6:6-7:1) ")\n" --> (7:6-10:34) ");\n\n//#endregion\n//# sourceMappingURL=main1.js.map"
+(2:0) "function " --> (4:0) "function "
+(2:9) "main() " --> (4:9) "main() "
+(2:16) "{\n" --> (4:16) "{\n"
+(3:2) "shared();\n" --> (5:0) "\t"
+(0:0) "import {shared} from './shared.js'\n" --> (5:1) "require_shared."
+(3:2) "shared()" --> (5:16) "shared()"
```

Comment:
This mapping now matches closer to evanw's viz [link](https://evanw.github.io/source-map-visualization/#MTgyACJ1c2Ugc3RyaWN0IjsKY29uc3QgcmVxdWlyZV9zaGFyZWQgPSByZXF1aXJlKCcuL3NoYXJlZC5qcycpOwoKLy8jcmVnaW9uIG1haW4xLmpzCmZ1bmN0aW9uIG1haW4oKSB7CglyZXF1aXJlX3NoYXJlZC5zaGFyZWQoKTsKfQptYWluKCk7CgovLyNlbmRyZWdpb24KLy8jIHNvdXJjZU1hcHBpbmdVUkw9bWFpbjEuanMubWFwMjQzAHsidmVyc2lvbiI6MywiZmlsZSI6Im1haW4xLmpzIiwibmFtZXMiOlsiIl0sInNvdXJjZXMiOlsiLi4vbWFpbjEuanMiXSwic291cmNlc0NvbnRlbnQiOlsiaW1wb3J0IHtzaGFyZWR9IGZyb20gJy4vc2hhcmVkLmpzJ1xuXG5mdW5jdGlvbiBtYWluKCkge1xuICBzaGFyZWQoKTtcbn1cblxubWFpbigpXG4iXSwibWFwcGluZ3MiOiI7Ozs7QUFFQSxTQUFTLE9BQU87QUFDZCxDQUhGQSxlQUdFLFFBQVE7QUFDVDtBQUVELE1BQU0ifQ==)

![image](https://github.com/user-attachments/assets/f3697f37-3f59-4e3d-884b-2861068902c2)